tornado_sqlalchemy/__init__.py

```diff
@@ -148,18 +126,180 @@ def _make_session(self) -> Session:
 set_max_workers = _async_exec.set_max_workers
 
 
-def make_session_factory(*args, **kwargs):
-    return SessionFactory(*args, **kwargs)
+class SessionEx(Session):
+    """The SessionEx extends the default session system with bind selection.
+    """
+
+    def __init__(self, db, autocommit=False, autoflush=True, **options):
+        self.app = db.get_app()
+        bind = options.pop('bind', None) or db.engine
+        binds = options.pop('binds', db.get_binds())
+
+        Session.__init__(
```

Comment:
Is there a specific reason why you chose `Session.__init__(self, ...)` instead of `super(...)`?